databases/pgcli/Portfile

```diff
@@ -16,20 +16,23 @@ long_description    ${description}
 
 homepage            http://pgcli.com
 
-checksums           rmd160  922f2cf729ab173edef221f39005d9e2d4d1c81c \
-                    sha256  e42291b45e762176218e2e937e7d1603f6f8eb32d17fd97907bb625d003e93dd
+checksums           rmd160  61fbaba17dd699f0fa0faf1a042a4525c7dead23 \
+                    sha256  75a5012a43462bf12d48baeb6bf25378dc117f9380af837112c3cf95fd945ee3
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.